backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -148,6 +149,30 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     */
   lazy val commandDirectory: Path = jobPaths.callExecutionRoot
 
+  /**
+    * Returns the shell scripting for hard linking the directory results using ln.
+    *
+    * @param directoryFiles The directorys.
+    * @return The shell scripting.
+    */
+  def directoryScripts(directoryFiles: Traversable[WomUnlistedDirectory]): String =
+    directoryFiles map directoryScript mkString "\n"
+
+  /**
+    * Returns the shell scripting for hard linking a directory results using ln.
+    *
+    * @param unlistedDirectory The directory.
+    * @return The shell scripting.
+    */
+  def directoryScript(unlistedDirectory: WomUnlistedDirectory): String = {
+    val directoryPath = DirectoryFunctions.ensureUnslashed(unlistedDirectory.value)
+    val directoryList = directoryPath + ".list"
+
+    s"""|# list all the files that match the directory into a file called directory.list
+        |find $directoryPath -type f > $directoryList
```

Comment:
`find` dependency 😮

Reply:
`find` is in [POSIX](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/find.html) so it's still available in docker images for busybox (and hence alpine), etc.

But if we attempt to use [`globstar`](https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html) to work around deeply nested directories, globstar would be bash only.

https://github.com/common-workflow-language/common-workflow-language/blob/7b831fe0b350af27dafb50ec89fd6411922fec39/v1.0/v1.0/iwdr_with_nested_dirs.cwl#L15-L20

Reply:
Fwiw I’m pretty sure someone using a bashless Docker is going to have a bad time anyways, unless that’s changed and I didn’t realize?

Reply:
I don't know what's coming in PAPIv2. If they remove their bash requirements, cromwell has exactly one non-POSIX requirement left: the string `bash` at the top of our `exec.sh` scripts.

On a UX front though there is the expectation of WDL authors that they can use bash-isms within their `command` blocks. But that could be mitigated w/ yet-another-config/runtimeAttribute/knob.

Reply:
I believe that’s gone with v2. Re the WDL that could go to being a wdl problem instead of a Cromwell problem